docs/troubleshooting-guide/visualization.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Troubleshooting question and dashboard visualizations
+---
+
+# Troubleshooting question and dashboard visualizations
+
+To start, check if your current browser settings are compatible with Metabase:
+
+1. Clear your browser cache, and refresh your page.
+2. Disable all extensions and plugins. Load the page again.
+3. Give it one last shot---try opening your page in an incognito session or a different browser.
+
+## Formatting dashboard cards
+
+1. Make sure that you're making and saving changes from the [card's settings](../dashboards/introduction.md#changing-a-cards-visualization-settings) (_not_ the original question's settings).
+2. [Reset your card's visualization settings](../dashboards/introduction.md#resetting-a-cards-visualization-settings).
+
+**Explanation**
+
+The visualization settings on a card are independent of the settings on the original question.
+
+When you first create a question, your selected visualization type is saved along with the query. When you add that question to a dashboard, the dashboard will display the same visualization as the original question by default. You can override the original visualization type by using the [card's visualization settings](../dashboards/introduction.md#changing-a-cards-visualization-settings).
+
+## Formatting dates and times
+
+See [Troubleshooting dates and times](./timezones.md).
+
+## Uploading a file for a custom map
+
+If you're unable to add a [GeoJSON file for a custom map](../configuring-metabase/custom-maps.md#custom-region-maps), try reducing the size of the file to 5MB or less.
```

Comment:
For these short sections that only really serve to reroute people elsewhere: do we need them? My concern is that they could muddy search results, which could put additional clicks between a search and a relevant answer.